app/src/main/java/com/ciandt/dragonfly/example/data/local/entities/VersionEntity.kt

```diff
@@ -40,6 +40,9 @@ data class VersionEntity(
         @ColumnInfo(name = "output_display_names")
         var outputDisplayNames: String = "",
 
+        @ColumnInfo(name = "closed_set")
+        var closedSet: String = "",
```

Comment:
Shoudln't this be a boolean instead?

Reply:
SQLite does not have a boolean type. We could use integer if it was a single value, but this is a comma-separated value.

Reply:
Got it.